eventful_django/eventful_tasks.py

```diff
@@ -13,10 +13,12 @@
 from google.cloud import pubsub_v1
 
 LOGGER = logging.getLogger(__name__)
+environ.pop("CELERY_BROKER_URL", None)
+environ.pop("CELERY_RESULT_BACKEND", None)
 
 CELERY_APP = Celery('eventful_tasks',
-                    backend=environ.get('EVENTFUL_BROKER_BACKEND', 'amqp'),
-                    broker=environ.get('EVENTFUL_BROKER_URL', 'amqp://localhost//'))
```

Comment:
I think we said we would fallback on BROKER evns if EVENTFUL evns are not set?

Reply:
You removed CELERY_BROKER_URL and removed EVENTFUL_BROKER_URL, so what is value of broker ?